geomodel.sh

```
@@ -3,77 +3,53 @@ version: "%(tag_basename)s"
+  - boost
   - Eigen3
+  - XercesC
+  - sqlite
+  - nlohmann-json
+  - hdf5
   - GEANT4
-  - opengl
-  - Coin3D
-  - SoQt
 build_requires:
   - CMake
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)